### DISCUSSION TITLE
HELP! Multiple keyboards in one configuration file for cheap split keyboard made out of NumPad keyboards

### DISCUSSION THREAD

**Discussion by asciible** [kmonad/kmonad#643] [General]
Would it be possible to use 2 input devices into one config file?

I would like to have a cheap "split keyboard" by using 2 super cheap NumPad keyboards in horizontal mode.
I know it is possible to have 2 running instances of kmonad controlling the specific output of each but I haven't figured out how to control the layers in one of the keyboards from the other one.
Just think about it, Numpad keyboards are extremely cheap, ortholinear and when in horizontal mode they offer multiple accessible thumb keys for layer manipulation.

**Comment by robrecord** [comment]
This is an amazing idea! Would love to see this...

**Comment by ArielHorwitz** [comment]
I've never tried running two instances of KMonad for different keyboards. However if that's possible, then can you make one instance trigger commands in another?

If so, I have an idea how to do this... But it may be not worth the effort 😅

Edit: I think [this](https://github.com/kmonad/kmonad/discussions/711) would be a more powerful and flexible solution but I don't think the authors are interested in such functionality. I've read them explicitly saying this software's purpose is to map keys and nothing more. Do one thing and do it well.

**Comment by ArielHorwitz** [comment]
So I found [this post](https://www.reddit.com/r/Kmonad/comments/18ubngp/input_merger_program_linux/) on reddit and I think it solves this.
